hack/vela-templates/cue/webservice.cue

```diff
@@ -39,8 +39,15 @@ output: {
 					ports: [{
 						containerPort: parameter.port
 					}]
+
+					resources: {
+						limits:
+							cpu: "1"
+						requests:
+							cpu: "0.1"
```

Comment:
make them optional as parameter instead of fixed value

Reply:
Sure.